app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(file_name: str) -> None:
+    words_of_file_name = file_name.split(" ")
+    old_file_name = words_of_file_name[1]
+    new_file_name = words_of_file_name[2]
```

Comment:
You should check that `words_of_file_name` contains 3 elements before unpacking value to avoid `IndexError`